binding/go/cheetah.go

```diff
@@ -105,6 +105,15 @@ var (
 	Version = nativeCheetah.nativeVersion()
 )
 
+// Returns a Cheetah struct with the given modelPath with default parameters
+func NewCheetah(accessKey string, modelPath string) Cheetah {
```

Comment:
I wonder if we should just even do just `AccessKey` for this constructor? It's meant to be the super-easy, batteries-included interface, so maybe we shouldn't confuse them with the empty string == default model.